script.favourites/changelog.txt

```diff
@@ -1,3 +1,9 @@
+v8.1.0
```

Comment:
Please use the <news> tag from the addon.xml for this and exclude this file.

Reply:
same here @ronie if possible